test/vm/containers/debianbuster.Dockerfile

```diff
@@ -0,0 +1,145 @@
+FROM docker.io/library/debian:buster
+
+LABEL maintainer="cncf-falco-dev@lists.cncf.io"
+# adopted from https://raw.githubusercontent.com/falcosecurity/driverkit/master/docker/builders/builder-any-x86_64_gcc8.0.0_gcc6.0.0_gcc5.0.0_gcc4.9.0_gcc4.8.0.Dockerfile
+
+ARG TARGETARCH
```

Comment:
I suppose we'd need a default value, otherwise all the deb packages won't have valid URL, like `https://download.falco.org/dependencies/cpp-6_6.3.0-18_${TARGETARCH}.deb`.
Experienced a build failure on Arch Linux and Docker client 24.0.2, where the resulting *cpp-6* package's URL is `https://download.falco.org/dependencies/cpp-6_6.3.0-18_.deb`, leading to an unexpected response body when running later `dpkg -i`. WDYT?